src/radha/Assigmnent_1_12_withoutTestNG/HandlingAlert1_TestCase3.java

```diff
@@ -0,0 +1,39 @@
+package radha.Assigmnent_1_12_withoutTestNG;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import base.PredefinedActions;
+
+public class HandlingAlert1_TestCase3 {
+
+	public static void main(String[] args) throws InterruptedException {
+		System.out.println("Launch browser");
```

Comment:
Ideally you write SOP statements after the action performed. other wise when script getting failed you always put debug point on wrong place.